py3status/modules/do_not_disturb.py

```diff
@@ -1,104 +1,225 @@
 # -*- coding: utf-8 -*-
 """
-Turn on and off dbus notifications.
+Toggle Do Not Disturb notifications.
 
-A left mouse click will toggle the state of this module.
+This module supports dunst, xfce4-notifyd, and lxqt-notificationd.
+For best results, don't install more than one notification server.
 
 Configuration parameters:
-    dunst_signal_off: Signal to send to the dunst process to disable "Do Not Disturb."
-        (default 'SIGTERM')
-    dunst_signal_on: Signal to send to the dunst process to enable "Do Not Disturb."
-        (default 'SIGUSR1')
-    format: Display format for the "Do Not Disturb" module.
-        (default '{state}')
-    notification_manager: The process name of your notification manager.
-        (default 'dunst')
-    refresh_interval: Refresh interval to use for killing notification manager process.
-        (default 0.25)
-    state_off: Message when the "Do Not Disturb" mode is disabled.
-        (default 'OFF')
-    state_on: Message when the "Do Not Disturb" mode is enabled.
-        (default 'ON')
+    cache_timeout: refresh interval for this module; for xfce4-notifyd
+        (default 10)
+    format: display format for this module
+        (default '{server} [\?if=state&color=good DND|\?color=bad DND]')
+    server: specify notification server to use, otherwise auto
+        (default None)
+    signals: specify 2-tuples of off/on signals; for dunst
```

Comment:
this description is poor what is '2-tuples'

A tuple consisting of the signal name for turning off notification followed by the signal name for turning on...
this is poor too but nearer - I'm not sure of what the setting does just guessing

Reply:
```
MISCELLANEOUS
    Dunst can be paused by sending a notification with a summary of
    "DUNST_COMMAND_PAUSE" and resumed with a summary of
    "DUNST_COMMAND_RESUME".  Alternatively you can send SIGUSR1 and
    SIGUSR2 to pause and unpause respectively. For Example:

    killall -SIGUSR1 dunst # pause
    killall -SIGUSR2 dunst # resume

    When paused dunst will not display any notifications but keep all
    notifications in a queue.  This can for example be wrapped around
    a screen locker (i3lock, slock) to prevent flickering of
    notifications through the lock and to read all missed notifications
    after returning to the computer.
```

Reply:
does our user care?

surely the thing is to have a setting like `pause_on_dnd` (bool) and then we do the right thing - now maybe it will only work for dunst but maybe we can expand to other notifiers if they learn or we discover how to implement